### ISSUE TITLE
Select code to dialogue, Chinese input

### ISSUE THREAD

**Issue by goWrongWay** [continuedev/continue#3319]
> why,  This time it caused problems with Chinese input.<img width="413" alt="Image" src="https://github.com/user-attachments/assets/077899c3-b9e4-4da4-86ff-55cd3f392376" />
> 
> <img width="409" alt="Image" src="https://github.com/user-attachments/assets/6a8e4ef9-7ae3-41c5-bdd8-0d7191edca0e" />
> 
>  

 _Originally posted by @goWrongWay in [b4d364f](https://github.com/continuedev/continue/commit/b4d364f7616a7bed85469d8655ff048b108aa786#r150215329)_

**Comment by Patrick-Erichsen** [comment]
Hi @goWrongWay , are you able to verify if changing that back to `p` rather than `div` allows you to input chinese characters?

**Comment by goWrongWay** [comment]
Yes, I can confirm that changing it back to P can input Chinese. Can you confirm it?

**Comment by felix1982** [comment]
I encountered the same problem.

**Comment by goWrongWay** [comment]
In fact, it has other problems. For example, in the vscode cmd+i editing state, Chinese cannot be input directly. I'm going to take a look now

**Comment by pnoker** [comment]
+1
